app/service/reviewer_notification_service.rb

```diff
@@ -0,0 +1,17 @@
+class ReviewerNotificationService
```

Comment:
what is the meaning of class_service.rb file name in services folder?
i mean **_service**